[INFO -- 2018-10-10 16:06:23]
Starting flip_value script
version: 1.1
[INFO -- 2018-10-10 16:06:23]
Initialization strings:
[INFO -- 2018-10-10 16:06:23]
(1) file /tmp/lreg/lreg
[INFO -- 2018-10-10 16:06:23]
(2) set args /tmp/lreg/randInput /tmp/lreg/output
[INFO -- 2018-10-10 16:06:23]
initSignal:0
[INFO -- 2018-10-10 16:06:23]
endSignal:10
[INFO -- 2018-10-10 16:06:23]
faultModel:2
[EXCEPTION -- 2018-10-10 16:06:24]
pythonException: empty range for randrange() (0,0, 0)
[EXCEPTION -- 2018-10-10 16:06:24]
pythonException: Unable to exit loop in struct fields; Exiting wihtout making a bit flip
[INFO -- 2018-10-10 16:06:24]
Backtrace BEGIN:
#0  __GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:51
#1  0x00007ffff6ec1801 in __GI_abort () at abort.c:79
#2  0x00007ffff7ad38fb in ?? () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#3  0x00007ffff7ad9d3a in ?? () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#4  0x00007ffff7ad9d95 in std::terminate() () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#5  0x00007ffff7ad9fe8 in __cxa_throw () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#6  0x00007ffff7b02f26 in std::__throw_bad_alloc() () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#7  0x0000555555557b3c in __gnu_cxx::new_allocator<double>::allocate (this=0x7fffffffd890, __n=18446744073709551599) at /usr/include/c++/7/ext/new_allocator.h:102
#8  0x00005555555578d5 in std::allocator_traits<std::allocator<double> >::allocate (__a=..., __n=18446744073709551599) at /usr/include/c++/7/bits/alloc_traits.h:436
#9  0x00005555555573ac in std::_Vector_base<double, std::allocator<double> >::_M_allocate (this=0x7fffffffd890, __n=18446744073709551599) at /usr/include/c++/7/bits/stl_vector.h:172
#10 0x00005555555570cd in std::_Vector_base<double, std::allocator<double> >::_M_create_storage (this=0x7fffffffd890, __n=18446744073709551599) at /usr/include/c++/7/bits/stl_vector.h:187
#11 0x00005555555567bd in std::_Vector_base<double, std::allocator<double> >::_Vector_base (this=0x7fffffffd890, __n=18446744073709551599, __a=...) at /usr/include/c++/7/bits/stl_vector.h:138
#12 0x000055555555628f in std::vector<double, std::allocator<double> >::vector (this=0x7fffffffd890, __x=std::vector of length -16, capacity -16 = {...}) at /usr/include/c++/7/bits/stl_vector.h:328
#13 0x00005555555558e8 in train (iterations=60000, N=3, stopsWhenStable=true) at lreg.cpp:165
#14 0x0000555555555dba in main (argc=3, argv=0x7fffffffde68) at lreg.cpp:225

46	../sysdeps/unix/sysv/linux/raise.c: No such file or directory.

Backtrace END
Memory content before bitflip:0xef0xff0xff0xff0xff0xff0xff0xff
Fault Model: Random bit-flip
base address to flip value: 0x7fffffffd730
address max offset: 8
Memory content after  bitflip:0x6f0xe00x1f0xf40xef0x1f0xb60x05
frame name: std::_Vector_base<double, std::allocator<double> >::_M_allocate
symbol name: __n
symbol filename: /usr/include/c++/7/bits/stl_vector.h
symbol line: 169
value: 411551531404353647
value address: 0x7fffffffd730
Type: The type is an integer type.
Type sizeof: 8

[INFO -- 2018-10-10 16:06:24]
Fault Injection Successful after 1.9677653312683105s
[INFO -- 2018-10-10 16:06:25]
event type: exit
[EXCEPTION -- 2018-10-10 16:06:25]
exit code: no exit code available
